projects/aca-content/src/lib/components/sidenav/sidenav.component.scss

```diff
@@ -113,8 +114,18 @@
       width: 100%;
     }
 
-    &__actions:last-child {
-      border-bottom: 0;
+    &__actions {
+      &__item:has(span) {
+        padding: 0;
+
+        span {
+          pointer-events: auto;
```

Comment:
Can you please check the cursor when you hover over items, it's flickering for me (you can barely see it in the video)

https://github.com/Alfresco/alfresco-content-app/assets/84377976/ea64521c-cff7-454e-917b-9391594577ee